circle.go

```diff
@@ -150,3 +155,26 @@ func GetBuild(org string, project string, buildNum int) (*CircleBuild, error) {
 	err = d.Decode(&cb)
 	return &cb, err
 }
+
+func CancelBuild(org string, project string, buildNum int) (*CircleBuild, error) {
+	token, err := getToken(org)
+	if err != nil {
+		return nil, err
+	}
+	uri := getCancelUri(org, project, buildNum, token)
+	body, err := makeRequest("POST", uri)
+	if err != nil {
+		return nil, err
+	}
+	defer body.Close()
+	var r io.Reader
+	if os.Getenv("CIRCLE_DEBUG") == "true" {
+		r = io.TeeReader(body, os.Stdout)
+	} else {
+		r = body
+	}
+	d := json.NewDecoder(r)
+	var cb CircleBuild
```

Comment:
Does `cancel` return you the canceled build in the output? I guess that makes sense